src/types/Base.js

```diff
@@ -67,6 +73,8 @@ export default class Base extends EventEmitter {
               bot.removeListener('update', listener);
             }
           });
+
+          return void 0;
```

Comment:
Is this to escape the linter? Instead I think you should avoid returning `resolve()`, but rather:

```
resolve();
return
```

Reply:
Eslinter was complaining about inconsistent return point, hence needed to return something, otherwise as per your `resolve(); return;` I'll add a new commit per both your comments. Thanks!